### DISCUSSION TITLE
0.126 shortcut issue with Z13

### DISCUSSION THREAD

**Discussion by githubbbbby** [seerge/g-helper#1374] [General]
I just installed version 0.126 on my Z13. The shortcut buttons are all messed up, volume controls the brightness, backlight button put to hibernate and so on.
I revert back to 0.124 and it works fine.

**Comment by seerge** [comment]
@githubbbbby hello, I'm sorry that "buttons are all messed up", but I didn't change anything related to hotkeys in 0.126 :)

Volume controls are handled by windows and not by the app (you can quit g-helper and they will still work)

In what circumstances does it happen? 
Do you have fn-lock on ? 
Do you have asus services running ? 

Post a log as well.

**Comment by githubbbbby** [comment]
Thank you for the fast reply (and the wonderful app! :) ) 
It just happened after I closed v0.124 and opened 0.126. It stopped after I went back to 0.124.
Fn-Lock is on.
all asus services are stopped from the "extra" g-helper window. 
(edited) I don't know how to send a log. 

I tried again and all seems fine now with 0.126?? I will update here if it happens again. sorry for the trouble :)

Also (edited) If it helps, when I opened the "extra" window after I launched 0.126 for the first time I'm sure I remember  the "key binding" showed "M1, M2, M3" (maybe FN+M1...) instead of FN+F2.
